### PR TITLE
Simplify configuration for web and uikit apps

### PR DESCRIPTION
**Pull request by AlexeyTsvetkov**

### CODE REVIEW

File: examples/falling-balls-mpp/build.gradle.kts [hunk 1]
```diff
@@ -74,8 +62,6 @@ kotlin {
                     "-linker-option", "-framework", "-linker-option", "CoreText",
                     "-linker-option", "-framework", "-linker-option", "CoreGraphics"
                 )
-                // TODO: the current compose binary surprises LLVM, so disable checks for now.
-                freeCompilerArgs += "-Xdisable-phases=VerifyBitcode"
```
Comment (olonho) [inline examples/falling-balls-mpp/build.gradle.kts:78]: Is it indeed no longer needed?

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.experimental.dsl
+
+import javax.inject.Inject
```
Comment (olonho) [inline gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt:8]: Why we need `@Inject` here?

**Review by olonho — APPROVED** [review]
Looks cool!

File: examples/falling-balls-mpp/build.gradle.kts (hunk 1 above)
Comment (AlexeyTsvetkov) [reply examples/falling-balls-mpp/build.gradle.kts:78]: It is already set https://github.com/JetBrains/compose-jb/pull/1682/files/242d2bb9d2746421e15a5e8a47385417aa6a32a5#diff-7367572e12d1f72a6493a4ded4868e01806a56006fe6deb1742634d4b1ed53dcR164

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt (hunk 2 above)
Comment (AlexeyTsvetkov) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt:8]: So Gradle can inject `name` constructor argument

**Pull request merged**